HW_one/Task_1.py

```diff
@@ -0,0 +1,9 @@
+#Задача 2: Найдите сумму цифр трехзначного числа.
+print ("Введите трехзначное число")
+n = int(input())
+sum = 0
+while n > 0:
+    x = n % 10
+    sum = sum + x
```

Comment:
Это можно совместить
```suggestion
    sum += n % 10
```